src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -59,9 +60,9 @@ class PaginationObj(JsonObject):
     count = IntegerProperty()
     total = IntegerProperty()
     size = IntegerProperty()
-    search_after = StringProperty()
```

Comment:
Can you motivate this change for me?

Reply:
The `search_after` and `search_before` parameters are always the same data type as the field that is chosen to be the `sort` parameter. Since it's possible for the that field to be an integer or a string, the search parameters must be set as a `DefaultProperty`, the dynamic field/property type of a `JSONObject`.

Reply:
Makes sense. Thank you.